### PR TITLE
feat: set and clear personal description now appears in change log

### PR DESCRIPTION
**Pull request by djaiss**
It didn't work before, now it does.

It also appears on the Audit log in the Settings.

image

### CODE REVIEW

**Review by asbiin — APPROVED** [review]

File: app/Services/Contact/Contact/UpdateContact.php [hunk 1]
```diff
@@ -6,9 +6,14 @@
 use App\Services\BaseService;
 use App\Models\Contact\Contact;
 use App\Jobs\Avatars\GenerateDefaultAvatar;
+use App\Services\Contact\Description\SetPersonalDescription;
+use App\Services\Contact\Description\ClearPersonalDescription;
 
 class UpdateContact extends BaseService
 {
+    private array $data;
+    private Contact $contact;
```
Comment (asbiin) [inline app/Services/Contact/Contact/UpdateContact.php:15]: This is php 7.4 specific syntax.
If we do this, we'll have to update the minimum php version also.

File: tests/Unit/Services/Contact/Contact/UpdateContactTest.php [hunk 2]
```diff
@@ -91,9 +94,11 @@ public function it_fails_if_wrong_parameters_are_given()
     public function it_throws_an_exception_if_account_doesnt_exist()
     {
         $contact = factory(Contact::class)->create([]);
+        $user = factory(User::class)->create([]);
 
         $request = [
             'account_id' => 11111,
+            'author_id' => $user()->id,
```
Comment (asbiin) [inline tests/Unit/Services/Contact/Contact/UpdateContactTest.php:101]: fix:
```suggestion
            'author_id' => $user->id,
```

**Review by asbiin — COMMENTED** [review]
See my comment

File: app/Services/Contact/Contact/UpdateContact.php (hunk 1 above)
Comment (djaiss) [reply app/Services/Contact/Contact/UpdateContact.php:15]: Damn.
But this is soooo useful!

Can we imagine updating the minimum php version then?

Comment (asbiin) [reply app/Services/Contact/Contact/UpdateContact.php:15]: I'm totally for it yes.
Let's do this.

Comment (asbiin) [reply app/Services/Contact/Contact/UpdateContact.php:15]: See #4900

Comment (djaiss) [reply app/Services/Contact/Contact/UpdateContact.php:15]: Great, will wait for it to be merged then.

Comment (asbiin) [reply app/Services/Contact/Contact/UpdateContact.php:15]: done! You'll just have to fix a test ;)

**Pull request merged**